README.md

```diff
@@ -32,6 +32,8 @@ Your AI assistant should automatically use these tools for Microsoft-related top
 
 > "Show me the complete guide for implementing authentication in ASP.NET Core. **fetch full doc**"
 
+> "give me a detailed, runnable python code to do harms eval using azure ai foundry evaluation sdk **using microsoft_code_sample_search**"
```

Comment:
show me detailed, runnable python code sample to do harms eval using azure ai foundry evaluation sdk

Reply:
Thanks, updated.